server/routes/caseNotes/caseNotesPresenter.ts

```diff
@@ -0,0 +1,30 @@
+import { Page } from '../../models/pagination'
+import { CaseNote } from '../../models/caseNote'
+import Pagination from '../../utils/pagination/pagination'
+import DateUtils from '../../utils/dateUtils'
+
+interface CaseNotesTableRow {
+  sentAtDateTime: string
+  sentBy: string
+  subject: string
+  body: string
+}
+
+export default class CaseNotesPresenter {
+  public readonly pagination: Pagination
+
+  constructor(private caseNotes: Page<CaseNote>) {
+    this.pagination = new Pagination(caseNotes)
+  }
+
+  readonly tableHeadings: string[] = ['Details', 'Case notes']
+
+  readonly tableRows: CaseNotesTableRow[] = this.caseNotes.content.map(caseNote => {
+    return {
+      sentAtDateTime: DateUtils.formattedDateTime(caseNote.sentAt),
```

Comment:
this seems like a sensible method to use. however i really don't like the way the formatted date times look for case notes. I think they probably work really well for appointments, but i would much prefer to see the date first, then the time. it makes visually grepping the dates much easier i think. what does the original design have?

Reply:
The original has it as `Wednesday 16 April 2021 3:55pm`

However I'm following the guidance from 

https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates

and

https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#times

Reply:
Ok, good point - I'll reform the date

Reply:
Sorry, I deleted my previous comment but you must have seen it. 

I agree regarding formatting of the date. I'll make that change.

Reply:
I've added new commit for getting day of week into DateUtils.